BeVolt/App/src/main.c

```diff
@@ -14,8 +14,10 @@
 #include "SoC.h"
 #include "LED.h"
 #include "SysTick.h"
+#include "TIM2.h"
```

Comment:
This no longer exists after the file rename, so needs to be changed.
```suggestion
#include "StatusSync.h"
```